### DISCUSSION TITLE
Práctica de CSS - retro

### DISCUSSION THREAD

**Discussion by santi4o** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#398] [Show and tell]
Hola. Pues este es el segundo ejercicio que hago de clonar diseños, el primero fue de la página de iniciar búsqueda de Google 😆, en este he aplicado lo que he estado aprendiendo de CSS estas semanas. Sé que somos muchos como para que cada uno reciba retro personalizada de su código, pero cualquier comentario o sugerencia que quieran proporcionar es bienvenida 😄. Iré actualizando mi repo para agregar algunos efectos de hover que parece que el diseño tiene, también hay una sección que también parece que debe tener efectos de collapse, pero decidí no usar javascript para darle enfoque exclusivo a CSS. ¡Saludos compañeros! #myFirstDiscussionPost

![2022-03-13 (3)](https://user-images.githubusercontent.com/37028475/158099523-6ed85fb4-46d7-4065-9633-a34299ef9d59.png)

![2022-03-13 (4)](https://user-images.githubusercontent.com/37028475/158099541-2dbe8fb1-927d-4a18-8354-5ef18b56500d.png)

[Enlace a mi repo](https://github.com/santi4o/LaunchX-FrontendMission-CSS)

**Comment by JuanSalvador73** [comment]
Pues está muy bien el sitio, y precisamente la idea es que conforme uno avance en este programa uses esos conocimientos para ir agregando o mejorando tu sitio web. Aunque por ahora solo piden el puro HTML sin ser responsivo, es mejor implementar sino ahora mas adelante. 

Han dejado muy claro que todos estos trabajos nuestros van a ser tomados en cuenta mas adelante, entre mas pros mejor.

Saludos.
